multiplatform/consumer-rules.pro

```diff
@@ -0,0 +1 @@
+-keep class ch.dreipol.dreiattest.multiplatform.api.** { *; }
```

Comment:
Please move `Attestation` to it's own file/ package `dao` e.g.

Reply:
I moved it to the package `dto` so that also additional dto classes which are might added in the future are excluded from obfuscation.